core/arch/arm/plat-hikey/spi_test.c

```diff
@@ -77,7 +77,7 @@ void spi_test(void)
 	platform_pl022_data.clk_hz = SPI_CLK_HZ;
 	platform_pl022_data.speed_hz = SPI_10_KHZ;
 	platform_pl022_data.cs_gpio_pin = GPIO6_2;
-	platform_pl022_data.mode = SPI_MODE0;
+	platform_pl022_data.mode = 0;
```

Comment:
Mode0 is 0, mode1 is 1 and so on so I thought it was a bit redundant and could be simplified to look less cluttered. Would you rather bring back the defines?